contrib/README.md

```diff
@@ -0,0 +1,71 @@
+# Contrib test suite
+
+The contrib folder contains simple end-to-end scripts to test integration of `huggingface_hub` in downstream libraries. The main goal is to proactively notice breaking changes and deprecation warnings.
+
+## Add tests for a new library
+
+To add another contrib lib, one must:
+1. Create a subfolder with the lib name. Example: `./contrib/transformers`
+2. Create a `requirements.txt` file specific to this lib. Example `./contrib/transformers/requirements.txt`
+3. Implements tests for this lib. Example: `./contrib/transformers/test_push_to_hub.py`
+4. Edit `makefile` to add the lib to `CONTRIB_LIBS` variable. Example: `CONTRIB_LIBS := timm transformers`
+5. Edit `.github/workflows/contrib-tests.yml` to add the lib to `matrix.contrib` list. Example: `contrib: ["timm", "transformers"]`
```

Comment:
I'd eventually look into automating these two so that it's slightly less error-prone, but as said above: ***nitpick***

Reply:
Now done by `make style` and `make quality`. Good call to reduce contribution efforts.